docs/project-imports/dcr.md

```diff
@@ -0,0 +1,117 @@
+# Importing `dotcom-rendering` into `csnx`
+
+## Glossary
+
+- `dotcom-rendering` refers to the application that renders HTML for `frontend`
+- _"the `dotcom-rendering` repo"_ refers to the repo that contains `dotcom-rendering`, `apps-rendering` and `common-rendering` codebases
+
+## Intention
+
+We want to relocate `dotcom-rendering` in the `csnx` monorepo.
+
+### Why?
+
+- to make it easier for Guardian product teams to ship by reducing the friction of interdependency between `dotcom-rendering` and other Guardian projects (Source, Libs, consent, commercial code etc)
+- to reduce the time spent by dotcom on managing their infrastructure
+- to standardise process: _best practice by default_
+
+## Preparation
+
+### Turn the `dotcom-rendering` repo into a single `yarn` workspace
+
+the `dotcom-rendering` repo currently managed by two different package managers:
+
+1. A `yarn` workspace containing `dotcom-rendering` and `common-rendering`
+2. An NPM-managed package that contains the `apps-rendering` code.
+
+_n.b. `common-rendering` is only consumed by AR._
```

Comment:
Not entirely true, the live blogs are shared (`LiveLayout.tsx`, `KeyEventsContainer.tsx`, `LiveBlock.tsx`)

Reply:
I think most of `common-rendering` is actually used by DCR. The components that Max mentioned ^, but also `Accordion`, `Pagination` and the fixtures. There are only couple of things that are exclusively used by AR, like the `FigCaption` component and parts of the `editorialPalette` modules.

Reply:
that's good to know, thanks very much